cloudformation/stacks/AutoSpotting/template.yaml

```diff
@@ -3,12 +3,19 @@
   Description: "AutoSpotting: automated EC2 Spot market bidder integrated with AutoScaling"
   Parameters:
     AllowedInstanceTypes:
-      Default: ""
+      Default: "*"
       Description: >
         "Comma separated list of allowed instance types for spot, in case you
-        may want to limit it to a smaller set of instance types  (also support
-        globs). If unset, instances will be chosen by AutoSpotting's algorithm.
-        Example: 'm4.xlarge,r4.xlarge,m5.*'"
+        may want to limit it to a smaller set of instance types (supports
+        globs). Example: 'm4.xlarge,r4.xlarge,m5.*'. If using the default or
+        leaving it unset, instances will be chosen by AutoSpotting's instance
+        compatibility algorithm based on CPU, memory, disk, etc., basically
+        getting you che cheapest available instances that are at least as big as
```

Comment:
was like this in the current master, i simply merged it.
Anyway i am pushing the fix..

Reply:
Thanks!

Reply:
as i was saying, 
stackset creation works as expected, you can either deploy all regions at creation, or later.
And you do not need to "reupdate" the stackset.
The only requirements, specified in the parameter description too, is that main region must be added as first region to the stackset deploy regions list.

About working service:
i tested all single parts, but i am waiting that aws terminate an instance (it's running from 7 days!) to completely test it.

I need it to fully test the cloudwatch events rule InputTransformer.

I tested an identically InputTransformer using a trigger based on "instance in service" event rule, so that to test it i simply launched a new istance, and it worked as expected so i am confident that it will work for spot termination event too.

And i tested the regional lambda using a "forged" "test event" too.

All parts worked as expected.

As soon as i complete last test i will inform you.

Reply:
Thanks, I think it's enough, I will merge this now and we'll try it out in our environment next week.